api/envoy/api/v2/cds.proto

```diff
@@ -1,6 +1,8 @@
 syntax = "proto3";
 
 package envoy.api.v2;
+option java_package = "io.envoyproxy.envoy.api.v2";
```

Comment:
Out of curiosity, is there no way to have the Java `protoc` plugin take the organizational package prefix (in this case `'io.envoyproxy`) as a CLI arguement and prefix it automatically? It seems this would be a common case thing to want to do.